exporter/datadogexporter/utils/http.go

```diff
@@ -31,6 +31,9 @@ var (
 		"Content-Type":     "application/x-protobuf",
 		"Content-Encoding": "identity",
 	}
+	Flavor    = "opentelemetry-collector-contrib"
+	Version   = "0.13.1"
```

Comment:
Note: Ideally this would use `internal/version`'s `Version` but this is not possible as of now (see #1347)